Source/WTF/wtf/PlatformHave.h

```diff
@@ -848,6 +848,12 @@
 #define HAVE_ASC_WEBKIT_SPI 1
 #endif
 
+#if PLATFORM(IOS) \
+    || PLATFORM(MAC) \
+    || PLATFORM(VISION)
+#define HAVE_WEBAUTHN_MODERN 1
```

Comment:
This doesn't seem to be used in your patch. Was this replaced with `HAVE_WEBAUTHN_AS_MODERN`?